src/main/java/duke/Task.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+public class Task {
+
+        protected String description;
+        protected boolean isDone;
+
+        public Task(String description) {
+            this.description = description;
+            this.isDone = false;
+        }
+        /**
+         * getter for status icon.
+         *
+         */
+        public String getStatusIcon() {
+            return (isDone ? "X" : " "); // mark done task with X
+        }
+        /**
+        * getter for task icon.
+        *
+        */
+        public String getTaskIcon() {
+            return "T";
+        }
+
+        /**
+        * returns string representation of task.
+        *
+        */
+        public String ToString(){
```

Comment:
Would be good to change to `toString` instead of `ToString`. Take note that doing so would also mean overriding the default `toString` object method.